js/packages/phoenix-client/src/utils/urls.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Utility functions for constructing URLs to Phoenix resources
+ */
+
+/**
+ * Get the base URL for Phoenix web UI
+ * @param baseUrl The base URL of the Phoenix API
+ * @returns The base URL for the Phoenix web UI
+ */
+export function getWebBaseUrl(baseUrl: string): string {
+  return baseUrl.endsWith("/") ? baseUrl : baseUrl + "/";
+}
```

Comment:
you might just leverage built-ins

Reply:
https://www.paulgraham.com/foundermode.html

Reply:
Whoops this link @GeLi2001 https://developer.mozilla.org/en-US/docs/Web/API/URL/URL